Doc/whatsnew/3.8.rst

```diff
@@ -349,6 +349,9 @@ Other Language Changes
   is ``-1``, and a suitable power of that inverse for other negative exponents.
   (Contributed by Mark Dickinson in :issue:`36027`.)
 
+* When dictionary comprehensions are evaluated, the key is now evaluated before
```

Comment:
Please add also an entry for the bytecode changes. Authors of the bytecode proceeding libraries will read that section but may miss changes in this section.

Reply:
Will update the PR soon.